Classes/EventStore/EventStore.php

```diff
@@ -52,6 +55,23 @@ public function get(EventStreamFilterInterface $filter): EventStream
         return $eventStream;
     }
 
+    /**
+     * Returns the EventAndRawEvent instance that matches the given $eventIdentifier - or throws an EventNotFoundException if no matching event could be found
+     *
+     * @param string $eventIdentifier
+     * @return EventAndRawEvent
+     * @throws EventNotFoundException
+     */
+    public function getOne(string $eventIdentifier): EventAndRawEvent
+    {
+        $rawEvent = $this->storage->loadOne($eventIdentifier);
+        if ($rawEvent === null) {
+            throw new EventNotFoundException(sprintf('Event with id "%s" could not be found', $eventIdentifier), 1513170518);
+        }
+        // TODO centralize RawEvent->EventAndRawEvent conversion
+        return (new EventStream(new \ArrayIterator([$rawEvent])))->current();
```

Comment:
This is not beautiful but working around this currently requires two new dependencies and code duplication.
If this gets merged I'll prepare a centralized class for event conversion (`RawEvent`->`EventAndRawEvent`, `EventInterface` => `array` vice versa, ...)